hello-world/src/transform.py

```diff
@@ -0,0 +1,30 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+The entry point of the pipeline Python wheel
+"""
+
+
+from pyspark.context import SparkContext
+from pyspark.sql.session import SparkSession
+from pyspark.sql import DataFrame
+
+
+sc = SparkContext.getOrCreate()
```

Comment:
Same as above about `SparkContext` being less encouraged and reducing global variable use. The spark session is not actually needed here - but you may be putting it here as an example.